jerry-ext/include/jerryscript-ext/module.h

```diff
@@ -92,20 +92,27 @@ void jerryx_native_module_register (jerryx_native_module_t *module_p);
 void jerryx_native_module_unregister (jerryx_native_module_t *module_p);
 
 /**
- * Declare the JerryScript module resolver so that it may be added to an array of jerryx_module_resolver_t items and
- * thus passed to jerryx_module_resolve.
+ * Declare the structure for module resolvers.
  */
-bool jerryx_module_native_resolver (const jerry_char_t *name, jerry_value_t *result);
+typedef struct
+{
+  jerry_value_t (*get_canonical_name) (const jerry_value_t name);
```

Comment:
Missing comments

Reply:
I need to know for sure. Do we, or do we not add `/**<` comments after structure members and function arguments? How do we document function pointer structure members, since the formal parameters of the function signature also needs to be documented?